### PR TITLE
noGrammar preference and README

### LINKED ISSUE #89: Add 'xml.problems.noGrammar' preference, to indicate document won't be validated
expose a new `xml.problems.noGrammar` preference, with valid values being `ignore|hint|info|warning|error`. See https://github.com/angelozerr/lsp4xml/commit/8ae6fc6f01f2de473e76d21c631bf659937fbb7f.

- [x] update package.json
- [x] update readme
- [x] update wiki

### PR DESCRIPTION
**Pull request by NikolasKomonen**
Fixes #89

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ The following settings are supported:
   
 Since 0.3.0:
 * `xml.format.spaceBeforeEmptyCloseTag`: Set to `true` to insert space before the end of a self closing tag.
+* `xml.problems.noGrammar`: Set the type of warning for grammar constraint issues. Default is `hint`.
```
Comment (fbricon) [inline README.md]: Set the message severity when a document has no associated grammar. Default is `hint`.

File: package.json [hunk 2]
```diff
@@ -163,6 +163,18 @@
...
+        "xml.problems.noGrammar" : {
+          "type": [
+            "ignore",
+            "hint",
+            "info", 
+            "warning", 
+            "error"
+          ],
+          "default": "hint",
+          "description": "The level of severity of which to indicate gramma constraint issues.",
```
Comment (fbricon) [inline package.json]: The message severity when a document has no associated grammar.

**Comment by NikolasKomonen** [comment]
@fbricon done

File: package.json [hunk 3]
```diff
@@ -163,6 +163,18 @@
           "default": "-noverify -Xmx64M -XX:+UseG1GC -XX:+UseStringDeduplication",
           "description": "Specifies extra VM arguments used to launch the XML Language Server. Eg. use `-noverify -Xmx1G  -XX:+UseG1GC -XX:+UseStringDeduplication` to bypass class verification, increase the heap size to 1GB and enable String deduplication with the G1 Garbage collector",
           "scope": "window"
+        },
+        "xml.problems.noGrammar" : {
```
Comment (fbricon) [inline package.json]: If you had tested it you'd have seen you missed "type": "string" and  "enum": [...]

**Pull request merged**